apps/webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Starkbot website
+
+Manage your keys from that website.
+
+### Dev setup
+
+Install `npm` and `nodejs` packages according to your distribution.
+create a *.env* file and add the following lines.
```

Comment:
Be carefull, you should put these environment variables in global .env file. You must fetch global .env and not local .env and adapt your code to this.

Reply:
It will automatically be the case when you resolve package.json issue